### PR TITLE
Eb iteration3

### PR DESCRIPTION
**Pull request by ericberglund117**
PR Templates:

What’s this PR do?  
-Created delete individual card functionality without reloading the page
-Created favorite individual card functionality without reloading the page
-Updated HTML with meta tag and img class names
-Updated disable property to save button through CSS
-Clean up WET code, console logs, and CSS display bugs
 
Where should the reviewer start?  
-Index.js
 
How should this be manually tested?  
-Check favorite and delete functionality of cards.
 
Any background context you want to provide?  
-Removed enableButton class and used CSS disable property to add functionality to save button
 
What are the relevant tickets?  
-N/A
 
Screenshots (if appropriate)  
-N/A
 
Questions:

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -40,8 +41,8 @@ <h1 class="header-text">IdeaBox</h1>
       <section class="idea-cards">
         <article class="idea-card">
```
Comment (npdarrington) [inline index.html:42]: Let's remember to remove the pre-loaded cards that are built into the page now that the functionality for adding/removing/favorite cards has been added.

File: src/main.js [hunk 2]
```diff
@@ -39,19 +45,57 @@ function saveIdea(){
...
+    ideaCardsGrid.removeChild(target);
+    for(var i = 0; i < list.length; i++){
+      if (target.getAttribute("id") == list[i].id) {
+        list.splice(i , 1);
+      };
+    };
+  };
+};
+
+function toggleFavoriteCard(){
```
Comment (nathanielmillard) [inline src/main.js:64]: This is one of the heftier sections I worth reviewing but that I am still excited to see running how we'd like.

File: src/main.js [hunk 3]
```diff
@@ -72,12 +116,10 @@ function displayIdeaCards(){
 }
 
 function disableEnableButton(){
-  if (inputTitle.value !== '' && inputBody.value !== ''){
-    saveButton.disabled = false;
-    saveButton.classList.add('buttonEnabled');
-  } else {
+  if (!inputTitle.value.trim() || !inputBody.value.trim()){
```
Comment (npdarrington) [inline src/main.js:119]: This was really clever! Great job for refactoring this to clear out spaces from being added and the card button accepting it as input.

**Review by nathanielmillard — COMMENTED** [review]
Really great work! Lots of fun refactor to get things working just a little better, a little simpler, and with fewer lines of code. 
I am excited to see if we can find a way to shorten/streamline more a few bulkier sections, but I am still proud of the logic we were still able to get up and functioning.

**Review by npdarrington — APPROVED** [review]

**Pull request merged**